pkg/kubectl/cmd/create_deployment.go

```diff
@@ -60,9 +61,67 @@ func NewCmdCreateDeployment(f cmdutil.Factory, cmdOut, cmdErr io.Writer) *cobra.
 	return cmd
 }
 
-// CreateDeployment implements the behavior to run the create deployment command
-func CreateDeployment(f cmdutil.Factory, cmdOut, cmdErr io.Writer, cmd *cobra.Command, args []string) error {
-	name, err := NameFromCommandArgs(cmd, args)
+// Return the name of the old generator (v1beta1) if server does not support
```

Comment:
could you write the docs of functions follow this format :   `functionName  return the name of ....`